product_docs/docs/pgd/5/reference/testingandtuning.mdx

```diff
@@ -4,117 +4,116 @@ navTitle: Testing and tuning
 indexdepth: 2
 ---
 
-EDB Postgres Distributed has tools which help with testing and tuning of your PGD clusters. For background, read the [Testing and Tuning](../testingandtuning) section.
+EDB Postgres Distributed has tools that help with testing and tuning your PGD clusters. For background, see [Testing and tuning](../testingandtuning).
 
 
-## `pgd_bench`
+## pgd_bench
 
 ### Synopsis
 
-A benchmarking tool for PGD enhanced PostgreSQL.
+A benchmarking tool for PGD-enhanced PostgreSQL.
 
 ```shell
 pgd_bench [OPTION]... [DBNAME] [DBNAME2]
 ```
 
-`DBNAME` may be a conninfo string of the format:
+`DBNAME` can be a conninfo string of the format:
   `"host=10.1.1.2 user=postgres dbname=master"`
 
-Consult the [Testing and Tuning - Pgd_bench](../testingandtuning#pgd_bench) section for examples
-of `pgd_bench` options and usage.
+See [pgd_bench in Testing and tuning](../testingandtuning#pgd_bench) for examples
+of pgd_bench options and usage.
 
 ### Options
 
-`pgd_bench` specific options include:
+pgd_bench-specific options include the following.
 
 #### Setting mode
 
 `-m` or `--mode`
 
-Which can be set to `regular`, `camo`, or `failover`. It defaults to `regular`.
+The mode can be set to `regular`, `camo`, or `failover`. The default is `regular`.
 
-* regular &mdash; Only a single node is needed to run `pgd_bench`
-* camo &mdash; A second node must be specified to act as the CAMO-partner (CAMO should be set up)
-* failover &mdash; A second node must be specified to act as the failover.
+* `regular` &mdash; Only a single node is needed to run pgd_bench.
+* `camo` &mdash; A second node must be specified to act as the CAMO partner. (CAMO must be set up.)
+* `failover` &mdash; A second node must be specified to act as the failover.
 
-When using `-m failover`, an additional option `--retry` is available. This will
-instruct `pgd_bench` to retry transactions when there is a failover. The `--retry` 
-option is automatically enabled with `-m camo`.
+When using `-m failover`, an additional option `--retry` is available. This option
+instructs pgd_bench to retry transactions when there's a failover. The `--retry` 
+option is enabled with `-m camo`.
```

Comment:
I think automatically should be added back

Reply:
```suggestion
option is automatically enabled when `-m camo` is used.
```